projects/storefrontlib/shared/components/form/form-errors/form-errors.component.ts

```diff
@@ -115,6 +137,15 @@ export class FormErrorsComponent implements DoCheck {
     return { ...errorDetails, ...this.translationParams };
   }
 
+  setAriaLiveAttribute() {
```

Comment:
I wonder if we can also set this directly in the host binding rather than calling from constructor?